README.md

```diff
@@ -28,5 +28,223 @@
 
 # Triton Inference Server
 
-**NOTE: You are currently on the r21.03 branch which tracks stabilization
-towards the next release. This branch is not usable during stabilization.**
\ No newline at end of file
+[![License](https://img.shields.io/badge/License-BSD3-lightgrey.svg)](https://opensource.org/licenses/BSD-3-Clause)
+
+**LATEST RELEASE: You are currently on the main branch which tracks
```

Comment:
This is not the main branch right? I think the changes are going in within the latest release branch.